pxtlib/emitter/emitter.ts

```diff
@@ -306,7 +306,7 @@ namespace ts.pxtc {
         advanced?: boolean;
         deprecated?: boolean;
         useEnumVal?: boolean; // for conversion from typescript to blocks with enumVal
-
+        field? :  Object;
```

Comment:
This interface is for the attributes that are parsed from the comments on ts and cpp files. Instead of storing the field attribute here, can we instead just check for it on line 1061?